portiaui/app/routes/index.js

```diff
@@ -1,7 +1,14 @@
 import Ember from 'ember';
+import hasBrowserFeatures from '../utils/browser-features';
+
+function identity(x) { return x; }
 
 export default Ember.Route.extend({
     redirect() {
-        this.transitionTo('projects');
+        hasBrowserFeatures().then((features) => {
+            let hasFeatures = features.every(identity);
```

Comment:
the documentation for redirect doesn't suggest that it follows the promise semantics of model/afterModel, so let's return hasBrowserFeatures from model() so that the route waits for it to resolve, and just transitionTo in redirect() based on the resolved value